hooks/pre/31-mongo_storage_engine.rb

```diff
@@ -0,0 +1,68 @@
+def fail_and_exit(message)
+  Kafo::Helpers.log_and_say :error, message
+  kafo.class.exit 1
+end
+
+def migration
+  katello = Kafo::Helpers.module_enabled?(@kafo, 'katello')
+  export_dir = '/var/tmp/mongodb_engine_upgrade'
+  mongo_dir = '/var/lib/mongodb'
+  hiera_file = '/etc/foreman-installer/custom-hiera.yaml'
+  mongodb_backup = '/var/tmp/mongodb_backup'
```

Comment:
this and the `export_dir` setting mean you need `2× mongodb size` space in `/var/tmp`, right? can we ensure this is true *before* doing a potentially long migration that will fail? not sure we have helpers for this available already.

Reply:
@evgeni this is handled here: 
https://github.com/Katello/katello-installer/blob/3f23cfcd0caf1789b8a69ecf564eb123b6cb8cb6/hooks/pre_validations/31-mongo_storage_engine.rb#L8

Reply:
oooh, ok, I was blind then. carry on :)